src/common/httpClient/httpClient.ts

```diff
@@ -26,26 +23,22 @@ export class HttpClient implements BasicHttpClient {
     }
 
     if (this.shouldLogHttpRequestAndResponse) {
-      this.logger.debug(`Sending request. Url: ${request.url}, Method: ${request.method}`, [
-        {
-          params: config.params,
-          body: config.body,
-        },
-      ]);
+      console.log(
```

Comment:
`logger.debug` should be replaced with `console.debug` so we do not loose log level information